portfolio/src/main/webapp/index.html

```diff
@@ -109,6 +109,9 @@ <h2>Social Media</h2>
       <a href="https://www.twitter.com/sydney_beas" class="fa fa-twitter"></a>
       <a href="https://www.facebook.com/sydney.beas" class="fa fa-facebook"></a>
       <a href="https://www.snapchat.com/add/syd7892" class="fa fa-snapchat-ghost"></a>
+      <br>
```

Comment:
I'd recommend perhaps wrapping the button and "data-holder" below in another div, which will take care of bringing the content to the next line, and will let you add extra space in CSS if you'd like. Prefer that over `br`.